libs/ml/include/ml/ops/add.hpp

```diff
@@ -104,6 +104,25 @@ class Add : public fetch::ml::ops::Ops<T>
     return OpType::OP_ADD;
   }
 
+  virtual typename fetch::ml::ops::Ops<T>::ChargeAmount OpForwardCost(
```

Comment:
you don't need virtual here (unless you plan to have an op that inherits from add)

Reply:
right, it's a copypasta artifact